framework-core/src/main/java/org/daisy/pipeline/job/Index.java

```diff
@@ -0,0 +1,27 @@
+package org.daisy.pipeline.job;
+
+public class Index {
```

Comment:
"Index" might be a bit too generic of a name to be explicit. I'm not actually 100% sure what this represents. It's the index of the result when the port creates a sequence of results, right? Can you please add a little javadoc to make it explicit?